content/post/17-your-new-normal-and-the-futility-of-stress/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Your New Normal and the Futility of Stress"
+date: 2013-06-03T00:00:00-07:00
+draft: false
+categories:
+- entrepreneurship
+- zen
+- general
+
+---
+I had a very powerful business experience about 4 years ago that taught me a lesson that I didn’t fully understand until recently.
+
+I was the sole owner of a company that received a Letter of Intent from a publicly traded company to be acquired for around $1 million, all before I was even 30 years old. Sure sounds sexy, right?  Well, the catch was that we were not profitable at that time and of course the time between receiving a Letter of Intent and actually signing all the contracts, getting a check in the bank, and considering the acquisition a done deal can be anywhere from a few weeks to over a year.
+
+So we waited. Doing our best to not be too unprofitable and not incur too many legal expenses, but thrilled at the prospect of an acquisition that would instantly remove all the stress. I remember every day trying to act to the outside world like things were fine, but inside I felt incredibly stressed. Each day was a new opportunity to obsess just a little bit more about when the acquisition would finally close.
+
+And then finally it did. Except that it didn’t go through, it *fell* through.
+<!--more-->
+And here’s the weirdest most crazy part about it all. I was enormously relieved.
+
+Huh?  After months of freaking out, how could I possibly feel relief?  I realized that even though I had a new set of problems to deal with — how to pay the legal expenses, what the new strategy should be, where to go from here, etc. — I was at least free of the uncertainty of whether “acqisition falls through” would be my new situation, my new normal, so to speak.
+
+And that’s what this post is all about.
+
+We human beings — particularly Americans — tend to stress. A lot. That stress is often related to having too much to do, not meeting all our goals, or otherwise somehow falling short of what’s expected of us.
+
+I was speaking to someone from Southeast Asia once and asked him what surprised him most about America. He said that he was puzzled by the term “stress management.”  “I thought you manage things that are precious to you, like your family, your health, or your money. Why would you manage stress?”
+
+It’s a good question!  Why don’t we call it “stress elimination” instead of “we know you can’t get rid of it, but here’s how you can make it better.”  Is stress just an unvoidable part of life, or a socially accepted particular type of craziness?
+
+I originally got into entrepreneurship because I’ve always been excited about creating something with technology and I didn’t want to be limited by anything other than myself. Each entrepreneur and business person I’ve met as their own motivators for getting into business, but almost all of them deal with non-trivial levels of stress.
```

Comment:
```suggestion
I originally got into entrepreneurship because I’ve always been excited about creating something with technology and I didn’t want to be limited by anything other than myself. Each entrepreneur and business person I’ve met has their own motivators for getting into business, but almost all of them deal with non-trivial levels of stress.
```